### ISSUE TITLE
Add check to see if Islandora host is running and responding

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_workbench_desktop#10]
The workbench script performs this check but we probably should do it in desktop too so it's easier to display a nice warning to the user. We need to check both that Islandora is up and running and that the credientials provided in the config file are valid.

Idea here is that if Islandora is not responding, Workbench Desktop should not call the python script but should tell the user to investigate.

**Comment by mjordan** [comment]
I obviously don't understand how try/catch works in Javascript, even though it's supposed to work the same as in other languages. This function returns true when Islandora is not responding:

```javascript
/**
 * Returs true if we can connect and log into Islandora, false otherwise.
 *
 * Does not currently work.
 */
function ping_islandora(config) {
  var http = require('http');
  http.get('http://localhost:800', function (res) {
    // @todo: Check response code, etc.
  }).on('error', function(e) {
    return false;
  });
  // @todo: Log in to make sure that credentials are valid.

  console.log("DEBUG: Reached end of ping_islandora function.")
  return true;
}
```

If anyone can point out why this is, I'd love to know.

I'm merging the issue-10 branch into master since I hope all we need to do is get this function to work as intended.

**Comment by mjordan** [comment]
Resolved with 52b4d703fff848e531e7f97d90b349f214d683bd. Thanks a million @seth-shaw-unlv !
